hpms/src/main/java/gov/cms/ab2d/hpms/service/HPMSFetcherImpl.java

```diff
@@ -26,10 +29,17 @@ public HPMSFetcherImpl(HPMSAuthService authService) {
         this.authService = authService;
     }
 
+    @SuppressWarnings("PMD.UnusedPrivateMethod")
+    @PostConstruct
+    private void buildURI() {
+        organizationBaseUri = buildFullURI(hpmsBaseURI + "/api/cda/orgs/info");
```

Comment:
You can deduplicate here and extract to a constant, there's a constants file that we use for URL segments like this or just put it at class level.

Reply:
If a literal is used in more than one place, then refactoring to a constant (within this class) or a Constant class makes sense.

*hpmsBaseURI* changes between lower and higher environments, so that's going to remain variable.

I'm not fully following the ~duplication~ concern, please clarify.

Reply:
"/api/cda/" repeat on both lines and likely will be used in other places in the app, normally there's some sort of base path URL constant that's used in these sort of situations, you can see other parts of the code base where we have a constant for the base path that's the start of the URL.

Reply:
Fixed